config/runtime.exs

```diff
@@ -114,6 +114,13 @@ config :ex_aws,
     port: System.get_env("MINIO_PORT")
   ]
 
+config :ex_typesense,
+  api_key: System.get_env("TYPESENSE_API_KEY"),
+  host: System.get_env("TYPESENSE_HOST") || "localhost",
+  port: String.to_integer(System.get_env("TYPESENSE_PORT") || "8108"),
```

Comment:
Do we really need this `String.to_integer/1` here? The default value is a string and it seems to be working fine.

Reply:
well it wasnt locally working when used without the S.to_i , it was logging error